ca/certificate-authority.go

```diff
@@ -314,15 +316,24 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 		return emptyCert, err
 	}
 
-	serialDec, err := ca.DB.IncrementAndGetSerial(tx)
+	// Hack: CFSSL always sticks a 64-bit random number at the end of the
+	// serialSeq we provide, but we want 136 bits of random number, plus an 8-bit
+	// instance id prefix. For now, we generate the extra 72 bits of randomness
+	// ourselves. We should modify CFSSL to just allow the caller to provide a
+	// own full serial number if it wants, and do all the generation logic
+	// ourselves.
+	randomPart := big.NewInt(1)
+	// We want 72 bits of randomness.
+	randomPart = randomPart.Lsh(randomPart, 72)
+	randomPart, err = rand.Int(rand.Reader, randomPart)
```

Comment:
Discussed off line but this could be maybe clearer as a Read from rand.Reader of 72 bytes and then a call to `*Int.SetBytes`.